src/pages/_app.tsx

```diff
@@ -12,18 +17,50 @@ const PreviewProductSwitcher = dynamic(
   { ssr: false }
 )
 
+if (typeof window !== 'undefined' && process.env.AXE_ENABLED) {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const ReactDOM = require('react-dom')
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const axe = require('@axe-core/react')
+  axe(React, ReactDOM, 1000)
+}
+
+const { ConsentManager, openConsentManager } = createConsentManager({
+  preset: 'oss',
+})
+
 export default function App({ Component, pageProps, layoutProps }) {
   useAnchorLinkAnalytics()
 
   const Layout = Component.layout ?? BaseLayout
+  const currentProduct = pageProps.product || null
+
+  const allLayoutProps = {
+    ...pageProps.layoutProps,
+    ...layoutProps,
+  }
```

Comment:
Possible TODO: fixing this to be `pageProps.layoutProps` instead of merging the two? Probably for a follow-up PR to get the merge done.

Reply:
TODO added in https://github.com/hashicorp/dev-portal/pull/175/commits/269d1280f5665e7b0bec3e99c77387668ccc07f5